modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -234,6 +236,40 @@ namespace cv {
                     fused_layer_names.back() = last_layer;
                 }
 
+                void setSwish()
+                {
+                    cv::dnn::LayerParams activation_param;
+                    activation_param.type = "Swish";
+
+                    darknet::LayerParameter lp;
+                    std::string layer_name = cv::format("swish_%d", layer_id);
+                    lp.layer_name = layer_name;
+                    lp.layer_type = activation_param.type;
+                    lp.layerParams = activation_param;
+                    lp.bottom_indexes.push_back(last_layer);
+                    last_layer = layer_name;
+                    net->layers.push_back(lp);
+
+                    fused_layer_names.back() = last_layer;
+                }
+
+                void setLogistic()
```

Comment:
Can you combine `setLogistic()` with `setSwish()` to avoid code duplication?
For example, `setActivation(String type)`.

Reply:
I have merged ReLU also.